File: app/src/main/java/ru/santaev/clipboardtranslator/ui/util/SettingsItem.java
package ru.santaev.clipboardtranslator.ui.util;

import android.databinding.BaseObservable;
import android.databinding.Bindable;
import android.databinding.ObservableField;

public class SettingsItem extends BaseObservable {

    @Bindable
    public final ObservableField<String> title = new ObservableField<>();

    @Bindable
    public final ObservableField<String> subtitle = new ObservableField<>();

    public SettingsItem(String title, String subtitle) {
        this.title.set(title);
        this.subtitle.set(subtitle);
    }
}

File: app/src/main/java/ru/santaev/clipboardtranslator/ui/settings/SettingsItem.kt
package ru.santaev.clipboardtranslator.ui.settings

import androidx.databinding.BaseObservable
import androidx.databinding.Bindable
import androidx.databinding.ObservableField

open class SettingsItem(
        title: String,
        subtitle: String
) : BaseObservable() {

    @Bindable
    val title = ObservableField<String>()

    @Bindable
    val subtitle = ObservableField<String>()

    init {
        this.title.set(title)
        this.subtitle.set(subtitle)
    }
}
